spec/schemas/_common.yaml

```diff
@@ -802,11 +802,11 @@ components:
       type: string
     StringifiedEpochTimeUnitSeconds:
       description: |-
-        Some APIs will return values such as numbers also as a string (notably epoch timestamps). This behavior
-        is used to capture this behavior while keeping the semantics of the field type.
+        Certain APIs may return values, including numbers like epoch timestamps, as strings. This setting captures 
+        behavior while keeping the semantics of the field type.
 
         Depending on the target language, code generators can keep the union or remove it and leniently parse
-        strings to the target type.
+        strings to the target type. 
```

Comment:
Is "to" the right preposition here, or do we mean "of" or "into"?

Reply:
To is correct here.